src/algebra/ordered_field.lean

```diff
@@ -177,6 +177,18 @@ else (div_le_div_left (lt_of_le_of_ne ha (ne.symm ha0)) hb hc).2 h
 
 end linear_ordered_field
 
+namespace nat
+
+variables {α : Type*} [linear_ordered_field α]
+
+lemma inv_pos_of_nat [linear_ordered_field α] {n : ℕ}  : 0 < ((n : α) + 1)⁻¹  :=
+  inv_pos $ add_pos_of_nonneg_of_pos n.cast_nonneg zero_lt_one
```

Comment:
Proofs/definitions (whatever comes after `:=`) should start in the leftmost column. There are some other instances of this.